apps/omg_watcher/lib/omg_watcher/exit_processor/exit_info.ex

```diff
@@ -125,7 +125,8 @@ defmodule OMG.Watcher.ExitProcessor.ExitInfo do
       exiting_txbytes: exit_info.exiting_txbytes,
       is_active: exit_info.is_active,
       eth_height: exit_info.eth_height,
-      root_chain_txhash: exit_info.root_chain_txhash
+      # defaults value to nil if non-existent in the DB.
+      root_chain_txhash: Map.get(exit_info, :root_chain_txhash, nil)
```

Comment:
map get default behaviour is returning nil

Reply:
```
If default is not provided, nil is used.

```

Reply:
Yes. I left it intentionally for clarity - but I suppose the comment is sufficient.